QUARKUS-2994.md

```diff
@@ -14,7 +14,7 @@ JIRA link: https://issues.redhat.com/browse/QUARKUS-2994
 ### Impact on test suites and testing automation
 * Temurin JVM 21 and OpenJDK 21 need to be added as JDKs to Quarkus QE Jenkins
 * Temurin JVM 21 and OpenJDK 21 have to be added as JVMs to Polarion
-* Bare metal jobs in extended platform testing need to be extended to run with both Temurin 21 and OpenJDK 21.
+* Bare metal Linux jobs in extended platform testing need to be extended to run with both Temurin 21 and OpenJDK 21.
```

Comment:
This is not what is happening in the job MR now, right? The way I see it there's extension for Windows on bare metal test suite as well, with one JDK version running with Podman and the other JDK version running with docker.

I actually re-read this plan and I think it was more correct before this change, if maybe not explicit enough.

Reply:
Cool cool, that makes sense. Can you add info about Windows jobs too into this PR too, then? I think it would make the plan much clearer.

Reply:
Fair point. The info is added.